app/controllers/inventory_foods_controller.rb

```diff
@@ -0,0 +1,82 @@
+class InventoryFoodsController < ApplicationController
+  # before_action :set_inventory_food, only: %i[show edit update destroy]
+
+  # GET /inventory_foods or /inventory_foods.json
+  def index
+    @inventory_foods = InventoryFood.all
+  end
+
+  # GET /inventory_foods/1 or /inventory_foods/1.json
+  def show; end
+
+  # GET /inventory_foods/new
+  def new
+    @inventory_food = InventoryFood.new
+  end
+
+  # GET /inventory_foods/1/edit
+  def edit; end
+
+  # POST /inventory_foods or /inventory_foods.json
+  def create
+    @inventory_food = InventoryFood.new(food_id: params[:food_id], quantity: params[:quantity])
+    @inventory_food.inventory_id = params[:inventory_id]
+    @inventory_food.save
+    if @inventory_food.save
+      redirect_to inventory_path(params[:inventory_id]), notice: 'Food was successfully added.'
+    else
+      render :new, alert: 'Food was not created.'
+    end
+
+    # respond_to do |format|
+    #   if @inventory_food.save
+    #     format.html do
+    #       redirect_to inventory_food_url(@inventory_food), notice: 'Inventory food was successfully created.'
+    #     end
+    #     format.json { render :show, status: :created, location: @inventory_food }
+    #   else
+    #     format.html { render :new, status: :unprocessable_entity }
+    #     format.json { render json: @inventory_food.errors, status: :unprocessable_entity }
+    #   end
+    # end
```

Comment:
- [ ] let's remove commented code as it makes code hard to read.

Reply:
Following commits for the solution:

- https://github.com/Linushaddai99/Group-Recipe-app/pull/33/commits/6a573f9af5785fbcfe8398324428088a521b2dd9